.envrc.example

```diff
@@ -2,3 +2,6 @@ export ALLOYDB_DB="some-db"
 export ALLOYDB_USER="some-user"
 export ALLOYDB_PASS="some-password"
 export ALLOYDB_INSTANCE_NAME="projects/<PROJECT>/locations/<REGION>/clusters/<CLUSTER>/instances/<INSTANCE>"
+export ALLOYDB_INSTANCE_IP="some-IP-address"
+export ALLOYDB_IAM_USER="some-IAM-user"
```

Comment:
Let's make this some-user@my-project.iam. Same below, just so folks know the format.

Reply:
Updated.